lib/ecs_deploy/auto_scaler/service_config.rb

```diff
@@ -171,23 +171,45 @@ def update_service(difference)
           @logger.info "Service \"#{name}\" clears cooldown state"
         end
 
-        cl = client
         next_desired_count = [next_desired_count, max_task_count[level]].min
+        update_service_desired_count(next_desired_count)
+
+        @last_updated_at = Process.clock_gettime(Process::CLOCK_MONOTONIC, :second)
+        self.desired_count = next_desired_count
+        @logger.info "Update service \"#{name}\": desired_count -> #{next_desired_count}"
+      rescue => e
+        pp e
```

Comment:
debug code?

Reply:
I don’t know whether or not because I only keep the original code, but I think it should be removed.

Reply:
I don't remember inserting this line at all, but it seemed me who did it ... 🙇 
I've removed this line https://github.com/reproio/ecs_deploy/pull/50/commits/56363ad474aaa7ad7060d0563d62273d56c28a25